backend/src/database/dto.rs

```diff
@@ -0,0 +1,80 @@
+use chrono::NaiveDateTime;
```

Comment:
why don't you use the one from sqlx?

Reply:
It is the same type, and other types from sqlx did not work. I changed the import in: [13f30bb](https://github.com/KILTprotocol/attester/pull/1/commits/13f30bba06bccb97a1b0a1ae31544360cb582134)